server/src/routes/token.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, RequestHandler, Response } from 'express';
+import { jwt } from 'twilio';
+import { TwilioCredentials } from '../common/types';
+import { retrieveAuthentication } from '../middlewares/sample-auth';
+
+export function createTokenRoute(
+  twilioCredentials: TwilioCredentials
+): RequestHandler {
+  const { AccessToken, AccessToken: { VoiceGrant } } = jwt;
+  return function tokenRoute(_req: Request, res: Response) {
+    const authentication = retrieveAuthentication(res);
+
+    if (typeof authentication === 'undefined') {
+      res.status(403).send('Unauthenticated request.');
+      return;
+    }
```

Comment:
No need to change it now since we're removing auth later. But note for future, you seem to be duplicating this on every route. A better way is, your middleware should take care of this. That is, every request coming in goes through auth.